core/trino-main/src/main/java/io/trino/execution/scheduler/faulttolerant/OutputStatsEstimator.java

```diff
@@ -33,15 +33,23 @@ enum OutputStatsEstimateStatus {
         FINISHED,
         ESTIMATED_BY_PROGRESS,
         ESTIMATED_BY_SMALL_INPUT,
-        ESTIMATED_FOR_EAGER_PARENT
+        ESTIMATED_FOR_EAGER_PARENT,
```

Comment:
There is conflicting work from @dekimir to get rid of this enum.
So far it was only used as a key for some debug logging, so we decided (given my suggestion) to replace it with just `String`. 
If we now want to use it in some control flow logic - maybe we should revert the decision.

Can you work together on that?